lib/cfa/placer.rb

```diff
@@ -1,6 +1,20 @@
 module CFA
-  # allows to place element at the end of configuration. Default one.
-  class AppendPlacer
+  # Places a new {AugeasElement} into an {AugeasTree}.
+  # Children of this abstract class implement different ways **where**
+  # to place the entry.
+  class Placer
+    # @param  [AugeasTree] tree
+    # @return [AugeasElement,Hash] the new element; it is empty!
+    #   Note that the return value is actually a Hash; {AugeasElement}
+    #   documents its structure.
```

Comment:
`@abstract` tag should be used http://www.rubydoc.info/gems/yard/file/docs/Tags.md#abstract